### ISSUE TITLE
Add API for the range input type

### ISSUE THREAD

**Issue by zabil** [getgauge/taiko#1031]
Depends on #612

Add an API for working with the range input type for example

From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range

```
<div>
  <input type="range" id="volume" name="volume"
         min="0" max="11">
  <label for="volume">Volume</label>
</div>
```

```
range({id: 'volume'}).exists()
range({id: 'volume'}).select(10)
range({min: 0, max: 11}).exists()
range({min: 0, max: 11}).select(11)
range(near("Volume")).select(10)
range({min: 0, max: 11}).value()
```

Geeky info https://youtu.be/uMSV4OteqBE?t=78

**Comment by o-x-y-g-e-n** [comment]
I will separate this from `textBox` API.

**Comment by NivedhaSenthil** [comment]
Implemented via 6e8e23c
